src/kaleidoscope/plugin/MouseKeys/MouseWrapper.h

```diff
@@ -43,14 +43,13 @@ namespace plugin {
 
 class MouseWrapper_ {
  public:
-  MouseWrapper_(void);
+  MouseWrapper_() {}
 
-  static void begin(void);
   static void move(int8_t x, int8_t y);
   static void warp(uint8_t warp_cmd);
+  static void end_warping();
   static void reset_warping();
-  static void pressButton(uint8_t button);
```

Comment:
where did the pressButton and releaseButton methods go?

Reply:
They went into the HID driver. Since MouseKeys was the only user of it, I just changed the call-site to call the driver's appropriate method instead.